### ISSUE TITLE
Task `chp.insert-chp-egon100re` fails due to missing file

### ISSUE THREAD

**Issue by nesnoj** [openego/eGon-data#900]
CI run (SH mode, but will most likely occur in DE too) throws:
```
FileNotFoundError: [Errno 2] No such file or directory: b'/storage/eGon-data-sh/working-dir/data_bundle_egon_data/pypsa_eur_sec/2022-05-04-egondata-integration/postnetworks/elec_s_37_lv2.0__Co2L0-3H-T-H-B-I-dist1_2050.nc'
```
That seems to be related to #884 where you changed paths to `2022-07-26-egondata-integration`..

**Comment by IlkaCu** [comment]
This commit should fix the problem: ca726c278e1abb8c24a5c1019c8c6f3b5435606f

**Comment by nesnoj** [comment]
Worked out, thx
